package.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name":     "jshint",
-  "version":  "2.4.4",
```

Comment:
The current jshint version is 2.9.2 - https://github.com/jshint/jshint/blob/master/package.json#L3. Is the fluid fork required to be forked from 2.4.4?